Dockerfile.routefix

```diff
@@ -1,4 +1,6 @@
-FROM registry.access.redhat.com/ubi8/ubi
-RUN dnf -y update \
-  && dnf -y install iproute \
-  && dnf clean all
+FROM registry.redhat.io/rhosp-rhel8/openstack-openvswitch-base:16.1
```

Comment:
any value in hard coding to some recent openshift ovs container release?  Then this file can be removed and it saves us from having to push the image through to arosvc.azurecr.io.

Reply:
@jim-minter the challenge is direct registry.redhat.io will requires a pull secret which is optional for ARO clusters.  If we have another way to mirror the image and make it available I'm open to switching to that but this was quick and easy.

Reply:
if you were to find that, say, the sdn image of 4.6.17 worked for what you needed, you could refer to `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bcc1fb20f06f00829727cb46ff21e22103fd4c737fdcbbf2fab13121f31ebcbd` in your daemonset; the ImageContentSources would then make sure that this would be pulled from our ACR as usual

Reply:
There is still a disconnect between this file and `version.RouteFixImage` used by the daemonset